### PR TITLE
Remove unaligned  with clean script

### PR DESCRIPTION
**Pull request by Smeds**
Added the possibility to remove Unaligned folder from archive folder.

### CODE REVIEW

File: cleanSweStore.pl [hunk 1]
```diff
@@ -170,7 +176,19 @@ =head1 OPTIONS
 		print $LEFTONSWESTORE "$runfolder\t$key\n";		
 	}
 
-					
+	if($removeUnaligned) {
+		#Check if Unaligned folder exists
```
Comment (b97pla) [inline cleanSweStore.pl]: better to save the path to a variable and reuse it in the statements below, e.g. `my $unalignedPath = $swestorePath/20$year-$month/$runfolder/Unaligned`

**Comment by b97pla** [comment]
I just have one comment, otherwise looks good! Let me know when you want me to merge.

**Comment by Smeds** [comment]
I have changed the script now

**Comment by b97pla** [comment]
:+1:

**Pull request merged**